src/content_scripts/contentlib.js

```diff
@@ -1,6 +1,6 @@
 import browser from 'webextension-polyfill';
 
-async function queryElement(query, node) {
+function queryElement(query, node) {
```

Comment:
Why did you remove async?

Reply:
Because the function doesn't do anything that's async. https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector doesn't return a promise.

Reply:
Oh sorry, read the diff wrong. Makes sense.